dockerfiles/examples/azure/azhpcc/templates/_helpers.tpl

```diff
@@ -0,0 +1,248 @@
+{{/*
+Expand the name of the chart.
+*/}}
```

Comment:
There looks to be much more in here than I would expect?

Reply:
yes.  I'm not sure how this chart should relate to the global helm  charts.  Should it use a relative include to pick up the _helper from the main repo?

Reply:
(see  comments above)

Reply:
My feeling is that is should only be a setup for storage for Azure.
And similarly there'd be one to provide storage for GKE and AWS.

The HPCC helm values would only point to pre-existing PVC's, e.g. provisioned by one of the above, although they are just examples, and users may already have some they want to use.

This is the approach I have currently been using.

Reply:
I generally agree - the heml chart only configures the storage, and currently clones _helpers.tpl, but I suspect it should include it instead.

As well as the storage there should probably be an example values.xml that ties together the standard helm charts and the pvcs defined here, and also defines some sensible resource limits - so that it is trivial to start a demonstration cluster on the appropriate engine.